sormas-app/app/src/main/java/de/symeda/sormas/app/lbds/LbdsDtoHelper.java

```diff
@@ -0,0 +1,206 @@
+package de.symeda.sormas.app.lbds;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.googlecode.openbeans.IntrospectionException;
+import com.googlecode.openbeans.Introspector;
+import com.googlecode.openbeans.PropertyDescriptor;
+
+import de.symeda.sormas.api.EntityDto;
+import de.symeda.sormas.api.ReferenceDto;
+import de.symeda.sormas.api.caze.CaseClassification;
+import de.symeda.sormas.api.caze.CaseDataDto;
+import de.symeda.sormas.api.caze.CaseOrigin;
+import de.symeda.sormas.api.caze.CaseOutcome;
+import de.symeda.sormas.api.caze.InvestigationStatus;
+import de.symeda.sormas.api.contact.FollowUpStatus;
+import de.symeda.sormas.api.person.PersonDto;
+import de.symeda.sormas.api.utils.pseudonymization.PseudonymizableDto;
+import de.symeda.sormas.app.backend.caze.Case;
+import de.symeda.sormas.app.backend.caze.CaseDtoHelper;
+import de.symeda.sormas.app.backend.common.AbstractDomainObject;
+import de.symeda.sormas.app.backend.common.AdoDtoHelper;
+import de.symeda.sormas.app.backend.person.Person;
+import de.symeda.sormas.app.backend.person.PersonDtoHelper;
+
+public class LbdsDtoHelper {
+
+	private static final List<String> PROPERTIES_TO_SKIP = Arrays.asList(
+		"class",
+		"enrolledInExternalJournal",
+		PersonDto.PHONE,
+		"creationVersion",
+		// TODO: check if embedded Dtos are empty:
+		CaseDataDto.HOSPITALIZATION,
+		CaseDataDto.EPI_DATA,
+		CaseDataDto.SYMPTOMS,
+		CaseDataDto.THERAPY,
+		CaseDataDto.CLINICAL_COURSE,
+		CaseDataDto.MATERNAL_HISTORY,
+		CaseDataDto.PORT_HEALTH_INFO);
+
+	private static final List<String> LBDS_PROPERTIES_ENTITY_DTO = Arrays.asList(
+		EntityDto.UUID,
+		EntityDto.CHANGE_DATE,
+		EntityDto.CREATION_DATE,
+		// Necessary for deserialization:
+		PseudonymizableDto.PSEUDONYMIZED);
+
+	private static final List<String> LBDS_PROPERTIES_PERSON_DTO = Arrays.asList(
+		PersonDto.FIRST_NAME,
+		PersonDto.LAST_NAME,
+		PersonDto.SEX,
+		// Necessary for deserialization:
+		PersonDto.COVID_CODE_DELIVERED,
+		PersonDto.HAS_COVID_APP,
+		PersonDto.PHONE);
+
+	private static final List<String> LBDS_PROPERTIES_CASE_DATA_DTO = Arrays.asList(
+		CaseDataDto.PERSON,
+		CaseDataDto.DISEASE,
+		CaseDataDto.REPORT_DATE,
+		CaseDataDto.REPORTING_USER,
+		CaseDataDto.CASE_CLASSIFICATION,
+		CaseDataDto.INVESTIGATION_STATUS,
+		CaseDataDto.REGION,
+		CaseDataDto.DISTRICT,
+		CaseDataDto.FACILITY_TYPE,
+		CaseDataDto.HEALTH_FACILITY,
+		// Necessary for deserialization:
```

Comment:
The following list of properties should be treated generically: All of these are `boolean` primitives, which is the reason for them being listed here.

Reply:
done